File: lab4/lang-tests/test-2/tipado.scala
object HelloWorld {
   def main(args: Array[String]) {
      println("prueba de tipado fuerte o debil")
      
      var num : Int = 10
      println(num)
      
      var word : String = "hola!!"
      println(word)
      
      var result = word * num
      println(result)
      println("concluimos que scala es de tipado DEBIL")
   }
}

File: lab4/lang-tests/test-2/tipado.java
public class tipado {
    public static void main(String[] args) {

        int num = 10;
        System.out.println(num);

        String word = new String("hola!!");
        System.out.println(word);

        String word2 = new String("chau!!");
        System.out.println(word2);

        String result = word + num;
        System.out.println(result);

        System.out.println("word * num es error");
        System.out.println("word * word es error");
        System.out.println("word + num es valido : 'hola!!10'");
        System.out.println("word + word es valido: 'hola!!chau!!'");

        System.out.println("java es de tipado DEBIL");

    }
}